packages/editor-ui/src/App.vue

```diff
@@ -15,17 +15,43 @@
 </template>
 
 <script lang="ts">
-import Vue from 'vue';
 import Modals from '@/components/Modals.vue';
 import Telemetry from './components/Telemetry.vue';
 
-export default Vue.extend({
+import mixins from 'vue-typed-mixins';
+import { showMessage } from './components/mixins/showMessage';
+
+export default mixins(showMessage).extend({
 	name: 'App',
 	components: {
 		Modals,
 		Telemetry,
 	},
-	mounted() {
+	data() {
+		return {
+			loading: true,
+		};
+	},
+	methods: {
+		async initialize(): Promise<void> {
+			try {
+				await this.$store.dispatch('settings/getSettings');
+				this.loading = false;
+			} catch (e) {
+				this.$showToast({
+					title: 'Error connecting to n8n',
+					message:
+						'Could not connect to server. <a onclick="window.location.reload(false);">Refresh</a> to try again',
```

Comment:
these should be localized. the reason these were not localized in UM is because this was implemented before i18n was added